docs/kmesh_support.md

```diff
@@ -1,16 +1,32 @@
-### Kmesh-capable OS
+### Support status of Kmesh
 
-Kmesh requires kernel eBPF functionality and sizeable eBPF Instruction Sets, so Kmesh can only run on operating systems above a certain version.
+Kmesh requires kernel eBPF functionality and sizeable eBPF Instruction Sets, so Kmesh can only run on Kernel version above **5.10**.
 
-Kmesh has two different modes, `Kernel-Native Mode` and `Duel-Engine Mode`. Although there is no difference between the two modes in terms of the os kernel version required, we still described separately in the table below.
+Kmesh uses istiod as a control plane and therefore Kmesh has some dependencies on istio versions and kubernetes versions.
 
-| Mode | OS Version | Kernel Version | Release Path |
+Kmesh has two different modes, `Kernel-Native Mode` and `Duel-Engine Mode`. While there is no difference in the OS kernel version required for the two modes, the supported istio versions differ. Therefore we explain them separately.
+
+- **Kmesh Duel-Engine Mode:**
+
+| Version | Request Kernel Version | Supported Istio Version | Support Kubernetes Version |
+| :-------------: | :-------------: | :-------------: | :-------------: |
+| main | >5.10 | 1.22, 1.23, 1.24 | 1.26, 1.27, 1.28, 1.29, 1.30, 1.31 |
```

Comment:
>=5.10

Reply:
s/main/latest ? I am not sure how other project describe it

Reply:
istio ues `master` to describe it